MapboxVision/VisionManager/VisionManager.swift

```diff
@@ -271,8 +271,6 @@ public final class VisionManager {
         dataProvider?.start()
         startVideoStream()
         dependencies.native.start(self)
-        
-        sessionManager.startSession(interruptionInterval: operationMode.sessionInterval)
```

Comment:
We want to record unless we know exact country. So it probably should start on resume.